### DISCUSSION TITLE
zh/guide/drivers/Seafile

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#366] [Announcements]
# zh/guide/drivers/Seafile

Seafile 是以资料库为单位进行版本控制和同步的开源云盘。 使用本驱动可以挂载 Seafile 的资料库到 Alist，已支持访问加密资料库。 根文件夹路径 1、如果未设置资料库ID（RepoId），则该配置可指定要访问的资料库的名称与子路径，默认为根路径：/，表示展示所有资料库。假设你有一个资料库名为 example，如果你只想挂载该资料库的内容...

https://alist.nn.ci/zh/guide/drivers/Seafile.html

**Comment by YFEGL** [comment]
配置好后访问缺报如下的错，seafile服务器版本：9.0.16
failed get objs: failed to list objs: request failed: <!DOCTYPE html> <html lang="en"> <head> <title>Seafile Server</title> <meta http-equiv="Content-type" content="text/html; charset=utf-8" /> <meta name="keywords" content="File, Collaboration, Team, Organization" /> <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" /> <meta http-equiv="x-ua-compatible" content="ie=edge" /> <link rel="icon" href="/media/favicons/favicon.png" type="image/x-icon"> <link rel="apple-touch-icon" href="/media/favicons/favicon.png"> <link rel="stylesheet" type="text/css" href="/media/assets/css/bootstrap.min.b00faad199b5.css"/> <link rel="stylesheet" type="text/css" href="/media/css/seahub.css?t=1398068110" /> <link rel="stylesheet" type="text/css" href="/media/css/sf_font3/iconfont.css" /> </head> <body> <div id="wrapper" class="en d-flex fd-col h100"> <div id="header" class="d-flex"> <a href="/" id="logo"> <img src="/media/img/seafile-logo.png" title="Seafile Server" alt="logo" width="" height="32" /> </a> <span class="sf2-icon-menu side-nav-toggle hidden-md-up hide" title="Side Nav Menu" id="js-toggle-side-nav" aria-label="Side Nav Menu"></span> <div id="lang"> <a href="#" id="lang-context" data-lang="en">English <span class="icon-caret-down"></span></a> <div id="lang-context-selector" class="sf-popover hide"> <ul class="sf-popover-con"> <li><a href="/i18n/?lang=ca">Català</a></li> <li><a href="/i18n/?lang=cs">Čeština</a></li> <li><a href="/i18n/?lang=de">Deutsch</a></li> <li><a href="/i18n/?lang=en">English</a></li> <li><a href="/i18n/?lang=es">Español</a></li> <li><a href="/i18n/?lang=es-ar">Español de Argentina</a></li> <li><a href="/i18n/?lang=es-mx">Español de México</a></li> <li><a href="/i18n/?lang=fr">Français</a></li> <li><a href="/i18n/?lang=it">Italiano</a></li> <li><a href="/i18n/?lang=is">Íslenska</a></li> <li><a href="/i18n/?lang=lv">Latvian</a></li> <li><a href="/i18n/?lang=hu">Magyar</a></li> <li><a href="/i18n/?lang=nl">Nederlands</a></li> <li><a href="/i18n/?lang=pl">Polski</a></li> <li><a href="/i18n/?lang=pt-br">Portuguese, Brazil</a></li> <li><a href="/i18n/?lang=ru">Русский</a></li> <li><a href="/i18n/?lang=sl">Slovenian</a></li> <li><a href="/i18n/?lang=fi">Suomi</a></li> <li><a href="/i18n/?lang=sv">Svenska</a></li> <li><a href="/i18n/?lang=vi">Tiếng Việt</a></li> <li><a href="/i18n/?lang=tr">Türkçe</a></li> <li><a href="/i18n/?lang=uk">українська мова</a></li> <li><a href="/i18n/?lang=he">עברית</a></li> <li><a href="/i18n/?lang=ar">العربية</a></li> <li><a href="/i18n/?lang=el">ελληνικά</a></li> <li><a href="/i18n/?lang=th">ไทย</a></li> <li><a href="/i18n/?lang=ko">한국어</a></li> <li><a href="/i18n/?lang=ja">日本語</a></li> <li><a href="/i18n/?lang=zh-cn">简体中文</a></li> <li><a href="/i18n/?lang=zh-tw">繁體中文</a></li> </ul> </div> </div> </div> <div id="main" class="container-fluid w100 flex-auto ov-auto"> <div class="row"> <div id="main-panel" class="w100 ovhd"> <div class="text-panel"> <p>Sorry, but the requested page could not be found.</p> </div> </div> </div> </div> <div id="confirm-popup" class="hide"> <div id="confirm-con"></div> <button id="confirm-yes">Yes</button> <button class="simplemodal-close">No</button> </div> </div> <script type="text/javascript"> var SEAFILE_GLOBAL = { csrfCookieName: 'sfcsrftoken' }; var app = { ui : { currentDropdown: false, currentHighlightedItem: false, freezeItemHightlight: false } } </script> <script type="text/javascript" src="/media/assets/scripts/lib/jquery.min.dc5e7f18c8d3.js" id="jquery"></script> <script type="text/javascript" src="/media/assets/scripts/lib/jquery.simplemodal.55150926fcd1.js"></script> <script type="text/javascript" src="/media/js/jq.min.js"></script> <script type="text/javascript" src="/media/js/base.js?t=1536127546642"></script> <script type="text/javascript"> function prepareAjaxErrorMsg(xhr) { var error_msg; if (xhr.responseText) { var parsed_resp = JSON.parse(xhr.responseText); // use `HTMLescape` for msgs which contain variable like 'path' error_msg = HTMLescape(parsed_resp.error || parsed_resp.error_msg || parsed_resp.detail); } else { error_msg = gettext("Failed. Please check the network."); } return error_msg; } function ajaxErrorHandler(xhr, textStatus, errorThrown) { var error_msg = prepareAjaxErrorMsg(xhr); feedback(error_msg, 'error'); } (function() { var lang_context = $('#lang-context'), lang_selector = $('#lang-context-selector'); // for pages without '#lang', such as 'help' pages if ($('#lang').length == 0) { return; } lang_context.parent().css({'position':'relative'}); if ($('#header').is(':visible')) { // for login page lang_selector.css({ 'top': lang_context.position().top + lang_context.height() + 5, 'right': 0 }); } var setLangSelectorMaxHeight = function() { if ($('#header').is(':visible')) { // for login page $('.sf-popover-con', lang_selector).css({ 'max-height': $(window).height() - lang_selector.offset().top - 12 }); } }; $(window).on('resize', function() { if (lang_selector.is(':visible')) { setLangSelectorMaxHeight(); } }); lang_context.on('click', function() { lang_selector.toggleClass('hide'); if (lang_selector.is(':visible')) { setLangSelectorMaxHeight(); } return false; }); $(document).on('click', function(e) { var element = e.target || e.srcElement; if (element.id != 'lang-context-selector' && element.id != 'lang-context') { lang_selector.addClass('hide'); } }); })(); if ($('.side-nav').length) { $('#logo').addClass('hidden-sm-down'); $('#js-toggle-side-nav').removeClass('hide'); } $('#js-toggle-side-nav').on('click', function() { $('.side-nav').addClass('side-nav-shown'); $('').modal({ overlayClose: true, onClose: function() { $('.side-nav').removeClass('side-nav-shown'); $.modal.close(); }}); $('#simplemodal-container').css({'display':'none'}); return false; }); $('.js-close-side-nav').on('click', function() { $('.side-nav').removeClass('side-nav-shown'); return false; }); </script> </body> </html>

**Reply by YFEGL** [reply]
加上RepoId就能正常用了，但按文档来说应该是不填应该是可以直接访问的啊
